### PR TITLE
[develop] Update hashes of GSI and rrfs_utils

### LINKED ISSUE #548: Update GSI hash
## Description

GSI has been updated to use `ncdiag` from hpc-stack instead of shipping it with the repository.
SRW should load `ncdiag/1.0.0` in `srw_common` and make sure GSI builds correctly with it.
I have encountered some errors while triying to upgrade GSI hash because it is needed for RRFS_dev1 merge work.
If someone familiar with GSI can take a look at this, it would be very helpful!

## Solution

Update GSI hash to latest and make sure SRW builds GSI properly.

## Requirements**

None

## Acceptance Criteria (Definition of Done)

GSI hash is updated and it builds correctly.

### PR DESCRIPTION
**Pull request by BenjaminBlake-NOAA**
## DESCRIPTION OF CHANGES: 
This PR updates the hashes of the GSI and rrfs_utils in Externals.cfg to the versions currently used by the real-time RRFS runs.  The GSI version is #4afe6ed (committed on March 23) and the rrfs_utils version is #4ec1a33 (committed on April 14).  To compile the newer version of GSI, ncdiag and ncio must be loaded to build the code.  I have made the changes to get it working on WCOSS2 and Hera, but will need assistance on the other supported platforms.

### Type of change

- [X ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## TESTS CONDUCTED: 

- [X ] hera.intel
- [ ] orion.intel
- [ ] cheyenne.intel
- [ ] cheyenne.gnu
- [ ] gaea.intel
- [ ] jet.intel
- [X ] wcoss2.intel
- [ ] NOAA Cloud (indicate which platform)
- [ ] Jenkins
- [ ] fundamental test suite
- [ ] comprehensive tests (specify *which* if a subset was used)

## DEPENDENCIES:
None.

## DOCUMENTATION:
No updates needed for this PR.

## ISSUE: 
Fixes issue #548 

## CHECKLIST

- [X ] My code follows the style guidelines in the Contributor's Guide
- [X ] I have performed a self-review of my own code using the Code Reviewer's Guide
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [X ] My changes do not require updates to the documentation (explain).
- [X ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published

## LABELS (optional): 

A Code Manager needs to add the following labels to this PR: 
- [X ] Work In Progress
- [ ] bug
- [X ] enhancement
- [ ] documentation
- [ ] release
- [ ] high priority
- [ ] run_ci
- [ ] run_we2e_fundamental_tests
- [ ] run_we2e_comprehensive_tests
- [ ] Needs Cheyenne test 
- [ ] Needs Jet test 
- [ ] Needs Hera test 
- [ ] Needs Orion test 
- [X ] help wanted

## CONTRIBUTORS (optional): 

@christinaholtNOAA  has volunteered to test on Jet

### CODE REVIEW

**Comment by MichaelLueken** [comment]
@BenjaminBlake-NOAA @christinaholtNOAA I'm willing to assist with testing to make sure that the updated hashes of the GSI and RRFS_UTILS build on the various machines, but should we move forward with updating the GSI at this point?  With the Intel 2022 run issues still not addressed (will be addressed with PR [#571](https://github.com/NOAA-EMC/GSI/pull/571)) and issues with HDF5 versions other than 1.10.6, should we wait until these issues have been resolved, or move forward.  I'm fine with updating the hashes now, but they will likely need to be updated again before the SRW v3.0.0 release.

**Comment by christinaholtNOAA** [comment]
The updated hash earlier rather than later will allow us to identify any other potential issues that may arise in the WE2E testing process as we add those workflow capabilities.

**Comment by BenjaminBlake-NOAA** [comment]
@MichaelLueken  I agree with @christinaholtNOAA that it would be beneficial to update the GSI sooner if possible - thanks.

**Comment by christinaholtNOAA** [comment]
I am working on building the branch on Jet.

**Comment by MichaelLueken** [comment]
@BenjaminBlake-NOAA

Attempted to build the updated GSI and rrfs_utils on Cheyenne.  It ultimately failed because ncdiag/1.0.0 isn't available in the hpc-stack on the machine for either Intel or GNU compilers.  I have notified @natalie-perlin about the missing ncdiag/1.0.0 on the machine and to please add this at her earliest convenience.

The GSI and rrfs_utils built without issue on Orion and Gaea (using the updated modulefile from PR #799).

**Comment by BenjaminBlake-NOAA** [comment]
@MichaelLueken  Sounds good!  Once ncdiag is added to Cheyenne, it should work.

**Comment by MichaelLueken** [comment]
@BenjaminBlake-NOAA I fear I spoke too soon on Gaea.  It looks like it is failing during the linking step to build the GSI executable due to undefined references to `mkl`.

I'll also reach out to @natalie-perlin to see if this is something that she can correct as well.

**Comment by christinaholtNOAA** [comment]
Everything looks good for the jet build. I used the appropriate flags to build everything and GSI/rrfs_utils.

Thanks @BenjaminBlake-NOAA for taking care of this one, and @MichaelLueken and @natalie-perlin for helping with the support on the other platforms.

**Comment by natalie-perlin** [comment]
@BenjaminBlake-NOAA @MichaelLueken @christinaholtNOAA -
ncdiag/1.0.0 has been added to Cheyenne to both intel/2022.1 and gnu/10.1.0 compiler stacks
Verified that both ncdiag/1.0.0 and ncio/1.1.2 have been built for the following platforms/stacks:
[x] Cheyenne intel/2022.1
[x] Cheyenne gnu/10.1.0
[x] Hera intel2022.1.2
[x] Hera gnu-9.2
[x] Jet intel/2022.1.2
[x] Orion intel/2022.1.2
[x] Gaea intel-classic/2022.0.2

**Review by natalie-perlin — APPROVED** [review]

**Review by christinaholtNOAA — APPROVED** [review]

**Comment by MichaelLueken** [comment]
> @BenjaminBlake-NOAA @MichaelLueken @christinaholtNOAA - ncdiag/1.0.0 has been added to Cheyenne to both intel/2022.1 and gnu/10.1.0 compiler stacks Verified that both ncdiag/1.0.0 and ncio/1.1.2 have been built for the following platforms/stacks:
[x] Cheyenne intel/2022.1
[x] Cheyenne gnu/10.1.0
[x] Hera intel2022.1.2
[x] Hera gnu-9.2
[x] Jet intel/2022.1.2
[x] Orion intel/2022.1.2
[x] Gaea intel-classic/2022.0.2

@natalie-perlin Unfortunately, the GSI and RRFS_UTILS continue to fail to build on Cheyenne for GNU compilers because the SRW App uses Cheyenne `gnu/11.2.0`, not `gnu/10.1.0`.  Will we need to revert the version of the GNU compiler to `10.1.0` or would it be possible to add this to the `11.2.0` stack?

With respect to the fails on Gaea due to undefined references to `mkl`, I highly doubt that the GSI has ever been compiled or run on the machine (at least, not since the modulefile was created back in Sept 2018).  The issue on Gaea might be more problematic to nail down.

**Comment by MichaelLueken** [comment]
@BenjaminBlake-NOAA - @natalie-perlin has added ncdiag/1.0.0 to the gnu/11.2.0 hpc-stack on Cheyenne, so the GSI now successfully builds on Cheyenne.  However, it was noted that the rrfs_utils fails to build on Cheyenne GNU.  The following errors are encountered:

```
/glade/scratch/mlueken/GSI/sorc/rrfs_utl/ens_mean_recenter/ens_mean_recenter.f90:223:16:

  223 |            stop(333)
      |                1
Error: Blank required in STOP statement near (1)
/glade/scratch/mlueken/GSI/sorc/rrfs_utl/ens_mean_recenter/ens_mean_recenter.f90:271:16:

  271 |            stop(555)
      |                1
Error: Blank required in STOP statement near (1)
/glade/scratch/mlueken/GSI/sorc/rrfs_utl/ens_mean_recenter/ens_mean_recenter.f90:326:19:

  326 |               stop(444)
      |                   1
Error: Blank required in STOP statement near (1)
```

at which point, the build fails.  Given this failure, I went ahead and tested on Hera GNU and the rrfs_utils failed to compile, this time with the following error:

```
/scratch2/NAGAPE/epic/Michael.Lueken/GSI/sorc/rrfs_utl/baselib/module_ncio.f90:13:6:

   13 |   use netcdf
      |      1
Fatal Error: Cannot open module file 'netcdf.mod' for reading at (1): No such file or directory
compilation terminated.
```

How should we go about moving forward with this PR?  Since everything builds without issue using Intel compilers, move ahead and state that only Intel compilers are supported, or should the rrfs_utils repo be updated so that it can once again build using GNU compilers?

Also, the building of the GSI is still encountering issues on Gaea due to the mkl libraries.  It looks like CMake is pulling the *.a files, rather than the *.so files, leading to failure at the linking stage to build the executable.  Should we note that the GSI won't compile on Gaea as well?

**Comment by christinaholtNOAA** [comment]
@MichaelLueken  Do the GNU builds work with the existing GSI and rrfs_utils hashes? I don't know that we'd ever tried that before.

Also, it'd be helpful from a workflow development point to go ahead with the Intel support of these builds for preparing the workflow around them. The GNU support is not required for RRFS outside the SRW release. Perhaps we should bring this up at the release meeting tomorrow.

**Comment by MichaelLueken** [comment]
@christinaholtNOAA -

I was able to successfully build the GSI and rrfs_utils hashes from the current develop branch on Cheyenne GNU.  The rrfs_utils repo failed with the same error as noted above on Hera GNU.

As you have noted, for the sake of NCO's RRFS implementation, we just need to ensure that the GSI and rrfs_utils compile and run using the Intel compiler and that it works on WCOSS2, Hera, and Orion.  It should also be noted that I doubt that the GSI regression tests have been run using GNU executables since I left EMC, so even though it compiles, the likelihood that the GSI will successfully run using the GNU compiler is unlikely.  It would probably be a good idea to bring this up during tomorrow's release meeting.

**Comment by christopherwharrop-noaa** [comment]
I really think you should not merge this until the GNU build errors for rrfs_utl have been addressed.  The `stop` errors are very basic code errors that are easily fixed.  And the `use netcdf` error is probably just a cmake issue or a lack of a GNU NetCDF build.

**Comment by mark-a-potts** [comment]
> 

I dug into the issues with building rrfs_utils using GNU on Hera and am putting together a PR for those changes into the RRFS_UTILS repo. They involve some code changes though, so they will need to be vetted to make sure they don't break anything. That code is ancient and has F77 fixed formatting, implicit variable (non)declarations, and non-standard intrinsic function calls. Yikes! 

I'll leave it to @MichaelLueken to decide if this PR can be merged now or needs to wait for the rrfs_utils fix to get done.

**Comment by mark-a-potts** [comment]
I put in the draft pull request here -- https://github.com/NOAA-GSL/rrfs_utl/pull/29

**Comment by christopherwharrop-noaa** [comment]
@mark-a-potts - Thank you for putting that PR together.  I took a look at it and it looks great to me.  (I did not actually run any tests, only eyeball thus far).  I would prefer that your fixes go in to this PR before it is merged so that we don't commit known broken code to develop.  The fixes are pretty basic and are easily testable and shouldn't hold anything up.

**Comment by MichaelLueken** [comment]
I agree with @christopherwharrop-noaa on this.  While the current rrfs_utls doesn't work with GNU, if a PR with the necessary corrections is available, I think it would be in everyone's best interest to bring those changes in when updating the GSI and rrfs_utls hashes.  Thank you very much, @mark-a-potts, for finding the issues and quickly opening a PR with necessary corrections to ensure that the rrfs_utls compiles with GNU compilers.

**Comment by mark-a-potts** [comment]
One other wrinkle. The hpc-stack does not have all the required modules (missing openblas, at least). The unified-env for spack-stack does have everything needed to build. Should we update the build_hera_gnu.lua modulefile to use spack-stack or add openblas to hpc-stack?

**Comment by MichaelLueken** [comment]
@mark-a-potts - We can't move to spack-stack until the ufs-weather-model has done so.  In this case, we would need add openblas to hpc-stack.  Would this need to be for both Hera GNU and Cheyenne GNU, or just Hera GNU?

**Comment by mark-a-potts** [comment]
Anyone know if there are regression tests for rrfs_utils or will we just need to test with some SRW E2E tests?

**Comment by christinaholtNOAA** [comment]
There is no testing framework for rrfs_utils. I think it's typically tested in the RRFS_dev1 retros as many of the other executable changes are.

**Comment by mark-a-potts** [comment]
I think that openblas needs to be added to the modulefile for build_hera_gnu.lua. With that addition, the devbuild.sh script works with gsi on Hera.

**Comment by mark-a-potts** [comment]
The rrfs_utils PR is ready for review, but I am unsure how to go about testing it.

**Comment by christinaholtNOAA** [comment]
The RRFS Utils PR was just merged. The hash is now 5681d1c. @BenjaminBlake-NOAA would you be willing to bump to that version in this PR?

**Comment by BenjaminBlake-NOAA** [comment]
@christinaholtNOAA  Yes I can do that.  I will test the build on WCOSS2 and Hera then push the change to my feature branch.

**Comment by MichaelLueken** [comment]
@mark-a-potts @BenjaminBlake-NOAA On Cheyenne GNU, rrfs_utils is still failing with the following:

```
[ 21%] Building Fortran object adjust_soiltq/CMakeFiles/adjust_soiltq.exe.dir/module_bkio_fv3lam_parall.f90.o
/glade/scratch/mlueken/ufs-srweather-app/sorc/rrfs_utl/adjust_soiltq/module_bkio_fv3lam_parall.f90:400:16:

  400 |            stop(333)
      |                1
Error: Blank required in STOP statement near (1)
/glade/scratch/mlueken/ufs-srweather-app/sorc/rrfs_utl/adjust_soiltq/module_bkio_fv3lam_parall.f90:453:16:

  453 |            stop(333)
      |                1
Error: Blank required in STOP statement near (1)
/glade/scratch/mlueken/ufs-srweather-app/sorc/rrfs_utl/adjust_soiltq/module_bkio_fv3lam_parall.f90:493:16:

  493 |            stop(333)
      |                1
Error: Blank required in STOP statement near (1)
/glade/scratch/mlueken/ufs-srweather-app/sorc/rrfs_utl/adjust_soiltq/module_bkio_fv3lam_parall.f90:527:16:

  527 |            stop(333)
      |                1
Error: Blank required in STOP statement near (1)
```

The same issue is encountered on Hera GNU:

```
[ 73%] Building Fortran object adjust_soiltq/CMakeFiles/adjust_soiltq.exe.dir/module_bkio_fv3lam_parall.f90.o
/scratch2/NAGAPE/epic/Michael.Lueken/ufs-srweather-app/sorc/rrfs_utl/adjust_soiltq/module_bkio_fv3lam_parall.f90:400:15:

  400 |            stop(333)
      |               1
Error: Blank required in STOP statement near (1)
/scratch2/NAGAPE/epic/Michael.Lueken/ufs-srweather-app/sorc/rrfs_utl/adjust_soiltq/module_bkio_fv3lam_parall.f90:453:15:

  453 |            stop(333)
      |               1
Error: Blank required in STOP statement near (1)
/scratch2/NAGAPE/epic/Michael.Lueken/ufs-srweather-app/sorc/rrfs_utl/adjust_soiltq/module_bkio_fv3lam_parall.f90:493:15:

  493 |            stop(333)
      |               1
Error: Blank required in STOP statement near (1)
/scratch2/NAGAPE/epic/Michael.Lueken/ufs-srweather-app/sorc/rrfs_utl/adjust_soiltq/module_bkio_fv3lam_parall.f90:527:15:

  527 |            stop(333)
      |               1
Error: Blank required in STOP statement near (1)
```

**Comment by mark-a-potts** [comment]
> @mark-a-potts @BenjaminBlake-NOAA On Cheyenne GNU, rrfs_utils is still failing with the following:
> 
> ```
> [ 21%] Building Fortran object adjust_soiltq/CMakeFiles/adjust_soiltq.exe.dir/module_bkio_fv3lam_parall.f90.o
> /glade/scratch/mlueken/ufs-srweather-app/sorc/rrfs_utl/adjust_soiltq/module_bkio_fv3lam_parall.f90:400:16:
> 
>   400 |            stop(333)
>       |                1
> Error: Blank required in STOP statement near (1)
> /glade/scratch/mlueken/ufs-srweather-app/sorc/rrfs_utl/adjust_soiltq/module_bkio_fv3lam_parall.f90:453:16:
> 
>   453 |            stop(333)
>       |                1
> Error: Blank required in STOP statement near (1)
> /glade/scratch/mlueken/ufs-srweather-app/sorc/rrfs_utl/adjust_soiltq/module_bkio_fv3lam_parall.f90:493:16:
> 
>   493 |            stop(333)
>       |                1
> Error: Blank required in STOP statement near (1)
> /glade/scratch/mlueken/ufs-srweather-app/sorc/rrfs_utl/adjust_soiltq/module_bkio_fv3lam_parall.f90:527:16:
> 
>   527 |            stop(333)
>       |                1
> Error: Blank required in STOP statement near (1)
> ```
> 
> The same issue is encountered on Hera GNU:
> 
> ```
> [ 73%] Building Fortran object adjust_soiltq/CMakeFiles/adjust_soiltq.exe.dir/module_bkio_fv3lam_parall.f90.o
> /scratch2/NAGAPE/epic/Michael.Lueken/ufs-srweather-app/sorc/rrfs_utl/adjust_soiltq/module_bkio_fv3lam_parall.f90:400:15:
> 
>   400 |            stop(333)
>       |               1
> Error: Blank required in STOP statement near (1)
> /scratch2/NAGAPE/epic/Michael.Lueken/ufs-srweather-app/sorc/rrfs_utl/adjust_soiltq/module_bkio_fv3lam_parall.f90:453:15:
> 
>   453 |            stop(333)
>       |               1
> Error: Blank required in STOP statement near (1)
> /scratch2/NAGAPE/epic/Michael.Lueken/ufs-srweather-app/sorc/rrfs_utl/adjust_soiltq/module_bkio_fv3lam_parall.f90:493:15:
> 
>   493 |            stop(333)
>       |               1
> Error: Blank required in STOP statement near (1)
> /scratch2/NAGAPE/epic/Michael.Lueken/ufs-srweather-app/sorc/rrfs_utl/adjust_soiltq/module_bkio_fv3lam_parall.f90:527:15:
> 
>   527 |            stop(333)
>       |               1
> Error: Blank required in STOP statement near (1)
> ```

So, it looks like those commands (actually that whole file) got added to rrfs_utils after I put in my PR, so they didn't get caught as an issue with gnu. The solution is to change them from "stop(333)" to "stop 333" in adjust_soiltq/module_bkio_fv3lam_parall.f90. I have verified that this compiles with both Intel and gnu compilers on Hera. I can put in another PR to rrfs_utils to address this issue.

**Comment by MichaelLueken** [comment]
> So, it looks like those commands (actually that whole file) got added to rrfs_utils after I put in my PR, so they didn't get caught as an issue with gnu. The solution is to change them from "stop(333)" to "stop 333" in adjust_soiltq/module_bkio_fv3lam_parall.f90. I have verified that this compiles with both Intel and gnu compilers on Hera. I can put in another PR to rrfs_utils to address this issue.

@mark-a-potts If you have verified that it builds successfully on GNU compilers, please go ahead and create the PR to update the new file.  Thanks!

**Comment by mark-a-potts** [comment]
The new PR is here-- https://github.com/NOAA-GSL/rrfs_utl/pull/31

**Comment by mark-a-potts** [comment]
The rrfs_utils PR has been merged, so that should now compile with GNU. I believe that an openblas module will need to be added to the GNU modulefiles (on Hera, at least) in order to get the GSI to compile, however.

**Comment by MichaelLueken** [comment]
@mark-a-potts @BenjaminBlake-NOAA The new rrfs_utls hash - [6cacff5](https://github.com/NOAA-GSL/rrfs_utl/commit/6cacff5a4e7ec2cb63465e123f49db7a886e222a), was successfully built on both Hera and Cheyenne.  The GSI also built without issue on both systems.

@mark-a-potts Would we see issues related with `openblas` at compile or runtime?  I'm not seeing any issues with the compile of the GSI on either Hera or Cheyenne.  Now, I'm noting failures of the `process_obs` WE2E test on Hera, with the following messages:

`/scratch2/NAGAPE/epic/Michael.Lueken/ufs-srweather-app/exec/process_larccld.exe: error while loading shared libraries: libmkl_gf_lp64.so: cannot open shared object file: No such file or directory`
`/scratch2/NAGAPE/epic/Michael.Lueken/ufs-srweather-app/exec/process_NSSL_mosaic.exe: error while loading shared libraries: libmkl_gf_lp64.so: cannot open shared object file: No such file or directory`

Please note that these messages are seen with and without openblas loaded during code compilation.

The test can be ran using:

`./run_WE2E_tests.py -t=process_obs -m=hera -c=gnu -a=epic`

in the tests/WE2E directory.

**Comment by christopherwharrop-noaa** [comment]
@MichaelLueken - So, the Intel MKL libraries are linked when building `process_larccld.exe` and `process_NSSL_mosaic.exe` with GNU?  It appears that MKL is used instead of OpenBLAS and that's why loading it doesn't matter.

**Comment by mark-a-potts** [comment]
@MichaelLueken, when I had built the SRW with "./devbuild.sh --platform=hera --compiler=gnu all" earlier on Hera, I had compile errors in GSI due to missing Lapack/Blas libraries. I'll test that again, though.

**Comment by MichaelLueken** [comment]
@mark-a-potts I compiled without issue using:

`./devbuild.sh -p=hera -c=gnu gsi rrfs_utils`

I'll test with `./devbuild.sh -p=hera -c=gnu all`

**Comment by MichaelLueken** [comment]
@christopherwharrop-noaa I don't know about the rrfs_utl repository, but I know that the GSI is set up to always load the MKL libraries.  If the MKL libraries aren't found, then the GSI will load LAPACK in it's place.  It looks like the GSI build will need to change to allow OpenBLAS for GNU and MKL for Intel.

**Comment by natalie-perlin** [comment]
@MichaelLueken @mark-a-potts @christopherwharrop-noaa - 
Please note that GSI modulefiles for GNU need to have the new/EPIC stacks loaded and the line setting MKLROOT variable commented out.  Openblas is to be loaded, and is successfully found for building LAPACK, instead of the MKL libraries for LAPACK.
```

-- pushenv("MKLROOT", "/apps/oneapi/mkl/2022.0.2")
local openblas_ver=os.getenv("openblas_ver") or "0.3.23"
load(pathJoin("openblas", openblas_ver))

```
Working on regression tests on Hera/GNU (some pass) at the moment, and on other modulefiles and test builds for the PR:
 https://github.com/NOAA-EMC/GSI/pull/571 , and also GNU modulefiles to resolve https://github.com/NOAA-EMC/GSI/issues/582

**Comment by MichaelLueken** [comment]
@natalie-perlin The SRW App doesn't use the component modulefiles to build the separate components.  All components are built using `modulefiles/build_hera_gnu.lua` in the SRW App itself.  The suggested change to the GSI modulefile doesn't affect the SRW App's build.

**Comment by natalie-perlin** [comment]
@MichaelLueken -  
  I was just mentioning that the openblas may needs to be loaded for the GSI, or MKL library set via "MKLROOT" variable for intel compilers.

File: modulefiles/srw_common.lua
Comment (natalie-perlin) [inline modulefiles/srw_common.lua:30]: Recent updates to GSI require ncdiag/1.1.0 (added to all hpc-stacks on Hera gnu/intel, Jet, Orion, Cheyenne gnu/intel, Gaea
 on June 2, 2023)

**Review by MichaelLueken — APPROVED** [review]
@BenjaminBlake-NOAA Following the latest modifications to this PR, the GSI and rrfs_utils now compile using both Intel and GNU compilers on Hera and Cheyenne and the `process_obs` WE2E test runs using both Intel and GNU executables on Hera.

Thank you very much @mark-a-potts  and @natalie-perlin creating the PRs in the rrfs_utl repo to correct the GNU build issues and adding OpenBLAS to the HPC-Stack!

Approving this work now.

**Comment by MichaelLueken** [comment]
@BenjaminBlake-NOAA The Jenkins tests failed on:

Cheyenne GNU - the `nco_grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15_thompson_mynn_lam3km` test failed, but this is a known issue (#806) and will not hold thePR back.

The Jenkins tests can't run on Gaea because the SRW App can't be built on the machine at this time.

Hera Intel - the `grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta`, `grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2`, and `grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16` tests failed.  The `grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta` failure is a known issue (#805) and won't hold the PR back.  The `grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2` and `grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16` tests were rerun on Hera Intel and successfully passed.

Orion - Manually ran the Jenkins tests and all tests successfully passed.

Merging this PR now.

**Pull request merged**
